src/include/pgagroal.h

```diff
@@ -235,6 +250,7 @@ struct prometheus
 
    atomic_ulong server_error[NUMBER_OF_SERVERS]; /**< The number of errors for a server */
    atomic_ulong failed_servers;                  /**< The number of failed servers */
+   struct prometheus_connection prometheus_connections[MAX_NUMBER_OF_CONNECTIONS];  /**< The number of prometheus connections */
```

Comment:
This needs to be a FMA field, so we only use the memory that we need.

See how `struct connection[];` is initialized after `max_connections` is known.